app/Controller/PagesController.php

```diff
@@ -84,79 +102,82 @@ public function display17() {
-    $aaaaaaaaaa;
+    public function display()
+    {
+        $eeeeeeeeee;
+        $aaaaaaaaaa;
+        $bbbbbbbbbb;
+        $aaaaaaaaaa;
 
-    $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
-		$path = func_get_args();
+        $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
+        $path = func_get_args();
 
-		$count = count($path);
-		if (!$count) {
-			return $this->redirect('/');
-		}
-		$page = $subpage = $title_for_layout = null;
+        $count = count($path);
+        if (!$count) {
+            return $this->redirect('/');
+        }
+        $page = $subpage = $title_for_layout = null;
 
-		if (!empty($path[0])) {
-			$page = $path[0];
-		}
-		if (!empty($path[1])) {
-			$subpage = $path[1];
-		}
-		if (!empty($path[$count - 1])) {
-			$title_for_layout = Inflector::humanize($path[$count - 1]);
-		}
-		$this->set(compact('page', 'subpage', 'title_for_layout'));
+        if (!empty($path[0])) {
+            $page = $path[0];
+        }
+        if (!empty($path[1])) {
+            $subpage = $path[1];
+        }
+        if (!empty($path[$count - 1])) {
+            $title_for_layout = Inflector::humanize($path[$count - 1]);
+        }
+        $this->set(compact('page', 'subpage', 'title_for_layout'));
 
-		try {
-			$this->render(implode('/', $path));
-		} catch (MissingViewException $e) {
-			if (Configure::read('debug')) {
-				throw $e;
-			}
-			throw new NotFoundException();
-		}
-  }
+        try {
+            $this->render(implode('/', $path));
+        } catch (MissingViewException $e) {
+            if (Configure::read('debug')) {
+                throw $e;
+            }
+            throw new NotFoundException();
+        }
+    }
 
-  private $顧客名;
+    private $顧客名;
 
-  function set顧客名($v)
-  {
-    $this->顧客名 = $v;
-    return $this;
-  }
+    public function set顧客名($v)
+    {
+        $this->顧客名 = $v;
+        return $this;
+    }
 
-  function get顧客名()
-  {
-    return $this->顧客名;
-  }
+    public function get顧客名()
+    {
+        return $this->顧客名;
+    }
 }
 
 /**
  * CyclomaticComplexity
  */
 // Cyclomatic Complexity = 12
-class Foo {
-    public function example()  {
-        if ($a == $b)  {
+class Foo
+{
+    public function example()
+    {
+        if ($a == $b) {
             if ($a1 == $b1) {
                 fiddle();
-            } else if ($a2 == $b2) {
+            } elseif ($a2 == $b2) {
                 fiddle();
-            }  else {
+            } else {
                 fiddle();
             }
-        } else if ($c == $d) {
+        } elseif ($c == $d) {
             while ($c == $d) {
                 fiddle();
             }
-         } else if ($e == $f) {
+        } elseif ($e == $f) {
```

Comment:
__[staging][phpmd]__
* Avoid variables with short names like $e. Configured minimum length is 3.
* Avoid variables with short names like $f. Configured minimum length is 3.
* Avoid unused local variables such as '$e'.
* Avoid unused local variables such as '$f'.